src/common/src/catalog/column.rs

```diff
@@ -225,7 +225,7 @@ impl ColumnDesc {
     }
 
     pub fn from_field_without_column_id(field: &Field) -> Self {
-        Self::from_field_with_column_id(field, 0)
+        Self::from_field_with_column_id(field, ColumnId::placeholder().into())
     }
```

Comment:
🥵 Has this ever been persisted?

Reply:
After this PR it is clear that this function is only called by `src/frontend/src/handler/create_source/external_schema/` and the name emphasizes `without_column_id`.

Before this PR, this utility is actually unused ...